### PR TITLE
.coafile: Fix spelling mistake

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

**Comment by hemangsk** [comment]
ack a02f456

**Comment by jayvdb** [comment]
@gitmate-bot ff

File: inactive_issues/inactive_issues_scraper.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def run(issues):
 def inactive_issues_json(request):
     try:
         GH_TOKEN = get_api_key('GH')
-    except:
+    except Exception:
```
Comment (Makman2) [inline inactive_issues/inactive_issues_scraper.py]: Separate commit? :)

Comment (jayvdb) [reply inactive_issues/inactive_issues_scraper.py]: I normally am happy to include minor fixes in the 'upgrade coafile' patch, so we can see what was problematic at the time.

Comment (Makman2) [reply inactive_issues/inactive_issues_scraper.py]: Ah I forgot that Pycodestyle gets now enabled because the typo is fixed... Nvm :+1:

**Pull request merged**